roles/aap_setup_prepare/tasks/main.yml

```diff
@@ -0,0 +1,36 @@
+---
+# tasks file for aap_setup_prepare
+
+# we check first so that the installation can happen without root access
+- name: check if the absolutely necessary packages are installed
+  raw: "rpm -q {{ __aap_setup_prep_required_packages | join(' ') }}"  # noqa 303
+  register: __aap_setup_prep_packages_result
+  ignore_errors: yes
+  changed_when: false
+  check_mode: false  # run this command even in check mode
+
+- name: install absolutely necessary packages
+  yum:
```

Comment:
should we change this to dnf since AAP 2.1 requires RHEL8+

Reply:
we should likely do that, though, I don't think it matters ansible wise, under the hood its likely still dnf.

Reply:
yes I agree it doesn't change much other than it would prevent it from working on RHEL 7